src/App.tsx

```diff
@@ -16,48 +19,76 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isSortedBy, setIsSortedBy] = useState<string>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+  const visibleGoods = getPreparedGoods(
+    goodsFromServer,
+    isSortedBy,
+    isReversed,
+  );
+
+  const Alphabet = () => {
+    setIsSortedBy(SortType.ALPHABETICAL);
+  };
+
+  const Length = () => {
```

Comment:
`Alphabet `, `Length` are not the best names  for functions which do some action

the name should start from verb like `sortByAlphabet`, `sortByLength`...
the name shoud be clear and you have to understand what this function do from its name